parts/linux/cloud-init/artifacts/cgroup-stats.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -o errexit
+
+EVENTS_LOGGING_DIR=/var/log/azure/Microsoft.Azure.Extensions.CustomScript/events/
+EVENTS_FILE_NAME=$(date +%s%3N)
+CSE_STARTTIME=$(date)
+CSE_STARTTIME_FORMATTED=$(date +"%F %T.%3N")
+CSE_ENDTIME_FORMATTED=$(date +"%F %T.%3N")
+NODEPOOL_NAME=$(curl -s -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01" | jq -r '.compute.osProfile.computerName')
+VM_TYPE=$(kubectl describe node $NODEPOOL_NAME --kubeconfig var/lib/kubelet/kubeconfig | grep "beta.kubernetes.io/instance-type=" | cut -d '=' -f 2)
+
+memory_string=$( jq -n \
+--arg USER_SLICE_MEMORY "$(cat /sys/fs/cgroup/user.slice/memory.current)" \
+--arg AZURE_SLICE_MEMORY "$(cat /sys/fs/cgroup/azure.slice/memory.current)" \
+--arg KUBEPODS_SLICE_MEMORY "$(cat /sys/fs/cgroup/kubepods.slice/memory.current)" \
+--arg SYSTEM_SLICE_MEMORY "$(cat /sys/fs/cgroup/system.slice/memory.current)" \
+--arg CONTAINERD_MEMORY "$(cat /sys/fs/cgroup/system.slice/containerd.service/memory.current)" \
+--arg KUBELET_MEMORY "$(cat /sys/fs/cgroup/system.slice/kubelet.service/memory.current)" \
+--arg ALLOCATABLE_MEMORY "$(cat /sys/fs/cgroup/kubepods.slice/memory.max)" \
```

Comment:
as we discussed allocatable also removes eviction. can we rename this slightly for clarity? `kubepods_cgroup_memory_max`?